src/app/about/page.tsx

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+export default function About() {
```

Comment:
Not sure if that's something related to the autoformatting in VSCode, but I am going to suggest you a few common rules for writing code.

This is optional so please take each comment as a suggestion. 
If whenever you save the file the latest changes are automatically reverted, then we need to update the `prettier` config (the one responsable of code formatting) but that's something very easy.

Reply:
I thought that was from prettier, but actually he lets you add an empty row between imports and exports.